src/renderer/app/components/Sidebar/Sidebar.module.scss

```diff
@@ -0,0 +1,104 @@
+@import "../../../css/bootstrap.imports.scss";
+
+:global {
+  .playing #sidebar .custom-scroll {
+    height: calc(100% - 122px) !important;
+  }
+}
+
+.sidebar {
+  width: 260px;
+  flex-shrink: 0;
+  background: $black;
+  position: fixed;
+  height: 100%;
+  overflow: hidden;
+
+  h2 {
+    font-size: .8rem;
+    text-transform: uppercase;
+    padding: 10px 30px 0;
+    color: $brand-grey;
+    font-weight: bold;
+    margin-bottom: .9rem;
+  }
+}
+
+.drag_strip {
+  height: 23px;
+  -webkit-app-region: drag;
+  flex-shrink: 0;
+
+  :global .mac & {
+    height: 53px;
+  }
+}
+
+.sidebar_menu {
+  margin-right: 10px;
+
+  :global {
+    .navLink {
```

Comment:
These were originally converted from their bootstrap variants, but besides this css they don't really have the need to be anywhere. So they can be made as local classes as well.